lib/publiccloud/ec2.pm

```diff
@@ -17,13 +17,58 @@ use testapi;
 
 has ssh_key      => undef;
 has ssh_key_file => undef;
+has credentials  => undef;
```

Comment:
later it will be moved upper to provider , right ?

Reply:
Depends, each CSP has it's own format of credentials, that's why I placed it here.